src/ILVerify/src/ILImporter.StackValue.cs

```diff
@@ -194,7 +194,7 @@ bool IsAssignable(TypeDesc src, TypeDesc dst, bool allowSizeEquivalence = false)
             }
 
             if (dst.IsInterface || dst.IsArray)
-                throw new NotImplementedException();
+                throw new NotImplementedException($"{nameof(IsAssignable)} is only partially implemented.");
 
             // TODO: Other cases - variance, etc.
```

Comment:
@MichalStrehovsky Thx for the hint to `CastingHelper.cs`. Fixing this TODO was one of the next things on my roadmap.